pkg/env/node_scan.go

```diff
@@ -3,6 +3,16 @@ package env
 import "time"
 
 var (
-	// NodeRescanInterval will set the duration for when to fetch node inventory to be scanned for vulnerabilities
-	NodeRescanInterval = registerDurationSetting("ROX_NODE_RESCAN_INTERVAL", 4*time.Hour)
+	// NodeScanningInterval is the base value of the interval duration between node scans.
+	NodeScanningInterval = registerDurationSetting("ROX_NODE_SCANNING_INTERVAL", 4*time.Hour)
+
+	// NodeScanningIntervalDeviation is the percentage by which each interval
+	// duration between node scans will deviate from the base interval time. A value
+	// equal or lower than zero disables deviation.
+	NodeScanningIntervalDeviation = RegisterIntegerSetting("ROX_NODE_SCANNING_INTERVAL_DEVIATION", 10)
+
+	// NodeScanningMaxInitialWait is the maximum wait time before the first node
+	// scan, which is randomly generated. Set zero to disable the initial node
+	// scanning wait time.
+	NodeScanningMaxInitialWait = registerDurationSetting("ROX_NODE_SCANNING_MAX_INITIAL_WAIT", 5*time.Minute)
```

Comment:
Nit: Thinking out loud, should we maybe base this off of the rescan interval? 
Something like 5-10% of that interval by default? That way, if we tune the rescan interval, we cannot forget about the Max Initial Wait 😄

Reply:
The initial and rescanning intervals try to alleviate the load but at different times of the Compliance lifetime. It might be desirable to tune them differently because of that. Although we always want to keep them updated and current, I feel we sacrificing the flexibility for that is a worse tradeoff.